.rubocop.yml

```diff
@@ -1,8 +1,9 @@
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 3.0
```

Comment:
```suggestion

```

if you remove this line then rubocop will use .ruby-version (probably cleaner as we won't have to update this every time we bump ruby).

Reply:
Made this change, but we have to update rubocop because the current `1.13.0` doesn't support ruby version 3.1.2